README.md

```diff
@@ -24,8 +24,10 @@ importStylesheet(  'User:USERNAME/Gadgets/DeepCat.css' );
 
 The current official release of the gadget can be found on wikipedia.org 
 ```
-mw.loader.load( "//de.wikipedia.org/w/index.php?title=User:Christoph Fischer (WMDE)/Gadgets/DeepCat.js&action=raw&ctype=text/javascript" );
-mw.loader.load( "//de.wikipedia.org/w/index.php?title=User:Christoph Fischer (WMDE)/Gadgets/DeepCat.css&action=raw&ctype=text/css" , "text/css" );
+$.when( mw.loader.using( [ 'mediawiki.api.messages', 'mediawiki.jqueryMsg' ] ), $.ready ).done( function() {
+    mw.loader.load( "//de.wikipedia.org/w/index.php?title=User:Christoph Fischer (WMDE)/gadget-test.js&action=raw&ctype=text/javascript" );
```

Comment:
This change of URLs here  might not be intended, I suppose? This is probably a version working with recent changes to the gadget code but not the URL we want to announce?
Just leaving a comment as to my knowledge the URL is going to be changed anyway (as the gadget moves out of user namespace)